tests/performance_tests/performance_test_basic.py

```diff
@@ -27,9 +28,13 @@ def waitForEmptyBlocks(node):
             emptyBlocks = 0
     return node.getHeadBlockNum()
 
+appArgs=AppArgs()
+extraArgs = appArgs.add(flag="--target-tps", type=int, help="The target transfers per second to send during test. Default: 1000", default=1000)
```

Comment:
I think if you look at the `--help` output that the Python library is already displaying the default value independently of the help string so including it in the help string is redundant, here and in the next option.

Reply:
Addressed: [b8cddd0](https://github.com/AntelopeIO/leap/pull/160/commits/b8cddd079ef86d2069cbfb713a5ebac073a534dc)